api/Element.json

```diff
@@ -8048,12 +8048,38 @@
             "edge": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": true
-            },
-            "firefox_android": {
-              "version_added": true
-            },
+            "firefox": [
+              {
+                "version_added": "85",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.menuitem.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "8",
```

Comment:
Where did version 8 come from, anyway?

Reply:
I initially trusted the original data and have confirmed support in 8-84 (not in 7 or 85) for `HTMLMenuItemElement` and api.HTMLElement.contextMenu.

It's very hard to confirm the same for events, and actually it looks like the `onshow` properties were added in Firefox 9, not 8. I suspect that the event was supported in Firefox 8 and the event handler property was forgotten but quickly added.

My suggestion is to leave the data consistent but incorrect, if we change how events are represented it truly won't be worth the effort to figure this out, even if we somehow leave a trace leading back to this PR.